react-context/flow.txt

```diff
@@ -0,0 +1,16 @@
+User can sign up
+  User will be able to register an account to the database from the client side. Once the user has navigated to the sign up page,
+  they can input their own username and password. Once submitted, the data will be queried to the database where the username is
```

Comment:
The data does not go straight to the database though, don't forget about the server!